docs/dev/api/api-testing.md

```diff
@@ -0,0 +1,1790 @@
+                "on_success": false
+            }
+        }
+    }]
+}
+```
+
+</details>
+
+
+
+## Test Methods
+
+### List All Tests in a Project
+
+<details><summary><span className="api get">GET</span><code>/api-testing/rest/v4/&#123;hookId&#125;/tests</code></summary>
+<p/>
+
+Returns a list of all tests within a project.
+
+#### Parameters
+
+<table id="table-api">
+  <tbody>
+   <tr>
+    <td><code>hookId</code></td>
+    <td><p><small>| PATH | REQUIRED | STRING |</small></p><p>Your project's hook ID, which you can create and/or retrieve from your project's <strong>Webhooks</strong> tab.</p></td>
+  </tr>
+</tbody>
+</table>
+
+
+<Tabs
+groupId="dc-url"
+defaultValue="us"
+values={[
+{label: 'United States', value: 'us'},
+{label: 'Europe', value: 'eu'},
+]}>
+
+<TabItem value="us">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request GET 'https://api.us-west-1.saucelabs.com/api-testing/rest/v4/3e540e3f-50bd-4088-8c1b-97f1d1530f14/tests' | json_pp
+```
+
+</TabItem>
+
+<TabItem value="eu">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request GET 'https://api.eu-central-1.saucelabs.com/api-testing/rest/v4/3e540e3f-50bd-4088-8c1b-97f1d1530f14/tests' | json_pp
+```
+
+</TabItem>
+</Tabs>
+
+#### Responses
+
+<table id="table-api">
+<tbody>
+  <tr>
+    <td><code>200</code></td>
+    <td colSpan='2'>Success.</td>
+  </tr>
+</tbody>
+<tbody>
+  <tr>
+    <td><code>401</code></td>
+    <td colSpan='2'>Authentication error.</td>
+  </tr>
+</tbody>
+</table>
+
+```jsx title="Sample Response"
+[{
+    "id": "621ad2cefd17a5416b299e97",
+    "name": "List all dog breeds",
+    "description": "Returns a list of all dog breeds",
+    "lastModified": "2022-02-27T07:02:35Z",
+    "tags": ["dogs", "doggos"],
+    "user": {
+        "id": "21b27f2d2aaa4a5c88c8c19df25857d3",
+        "name": "first.last"
+    },
+    "complete": true,
+    "status": {
+        "success": true,
+        "lastUpdate": "2022-02-27T08:58:01Z"
+    },
+    "schedules": {
+        "total": 1,
+        "active": 1
+    }
+}, {
+    "id": "621b20e8fd17a5416b299e9d",
+    "name": "List all sub-breeds: retrievers",
+    "description": "Returns an array of all the sub-breeds from a breed",
+    "lastModified": "2022-02-27T07:05:25Z",
+    "tags": ["dogs", "retrievers"],
+    "user": {
+        "id": "21b27f2d2aaa4a5c88c8c19df25857d3",
+        "name": "first.last"
+    },
+    "complete": true,
+    "status": {
+        "success": true,
+        "lastUpdate": "2022-02-27T09:47:33Z"
+    },
+    "schedules": {
+        "total": 1,
+        "active": 1
+    }
+}]
+```
+
+</details>
+
+---
+
+### Upload a Test
+
+<details><summary><span className="api put">PUT</span><code>/api-testing/rest/v4/&#123;hookId&#125;/tests</code></summary>
+<p/>
+
+Uploads a new test or modifies the test if it already exists.
```

Comment:
@kimsauce maybe it's better "Uploads the test you send in the body into the cloud platform..."